indico/modules/events/templates/emails/move_request_closure.txt

```diff
@@ -0,0 +1,31 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    Event move request update
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% block body -%}
+    {%- filter dedent -%}
+        {{ ('The following events have been moved:' if accept else 'Move requests of the following events have been rejected:' ) }}
+
+        {% filter underline %}Events{% endfilter %}
+        {%- for evt in events %}
+            {{ evt.title }}
+            - URL: {{ evt.external_url }}{% if accept %}, New event path: {{ evt.category.chain_titles|join(' » ') }}{% endif %}
```

Comment:
just FYI, jinja supports trimming strings on any side of a tag, so you don't have to put so much in one line. e.g. `{%- if accept %}` would strip any whitespace before the `if` tag